monero-outreach-docs/translations/zh-hant/storyboard/wallet_quickstart.md

```diff
@@ -0,0 +1,57 @@
+# 門羅幣錢包快速上手
+*04/22/19*  
+Almutasim  
```

Comment:
a leftover, i guess

Reply:
Could you elaborate more? I thought `Almutasim` is just the author's name.

Reply:
IIRC Almutasim is the author of the Chinese translation, not of the document.

Reply:
But I copied the file from [en version](https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/en/storyboard/wallet_quickstart_en.md). It's Almutasim on it, or did I get anything wrong?

Reply:
I might be confused because he also contributed with the zh-cn translation. ignore my comment.

Reply:
I see.